api/s2n.h

```diff
@@ -999,6 +999,12 @@ S2N_API extern int s2n_config_set_verify_host_callback(struct s2n_config *config
  *
  * The default value is 1 if the underlying libCrypto implementation supports OCSP. 
  *
+ * @note SHA-1 is the only supported hash algorithm for the `certID` field. If a different hash 
+ * algorithm is used, validation will fail and manual verification will be needed. This is 
+ * different from the hash algorithm used for the signature algorithm. See 
+ * [RFC6960](https://datatracker.ietf.org/doc/html/rfc6960#section-4.1.1) for details about 
+ * `certID` field.
+ * 
```

Comment:
I think `s2n_config_set_status_request_type()` might be a better place to document this. If users were to care about this they probably would want to know when enabling OCSP.